specification/storage/data-plane/Microsoft.BlobStorage/preview/2019-02-02/blob.json

```diff
@@ -0,0 +1,10000 @@
+              }
+            }
+          },
+          "default": {
+            "description": "Failure",
+            "headers": {
+              "x-ms-error-code": {
+                "x-ms-client-name": "ErrorCode",
+                "type": "string"
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/StorageError"
+            }
+          }
+        }
+      },
+      "parameters": [
+        {
+          "name": "comp",
+          "in": "query",
+          "required": true,
+          "type": "string",
+          "enum": [
+            "lease"
+          ]
+        },
+        {
+          "name": "restype",
+          "in": "query",
+          "required": true,
+          "type": "string",
+          "enum": [
+            "container"
+          ]
+        },
+        {
+          "name": "x-ms-lease-action",
+          "x-ms-client-name": "action",
+          "in": "header",
+          "required": true,
+          "type": "string",
+          "enum": [
+            "change"
+          ],
+          "x-ms-enum": {
+            "name": "LeaseAction",
+            "modelAsString": false
+          },
+          "x-ms-parameter-location": "method",
+          "description": "Describes what lease action to take."
+        }
+      ]
+    },
+    "/{containerName}?restype=container&comp=list&flat": {
+      "get": {
+        "tags": [
+          "containers"
+        ],
+        "operationId": "Container_ListBlobFlatSegment",
+        "description": "[Update] The List Blobs operation returns a list of the blobs under the specified container",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Prefix"
+          },
+          {
+            "$ref": "#/parameters/Marker"
+          },
+          {
+            "$ref": "#/parameters/MaxResults"
+          },
+          {
+            "$ref": "#/parameters/ListBlobsInclude"
+          },
+          {
+            "$ref": "#/parameters/Timeout"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ClientRequestId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success.",
+            "headers": {
+              "Content-Type": {
+                "type": "string",
+                "description": "The media type of the body of the response. For List Blobs this is 'application/xml'"
+              },
+              "x-ms-client-request-id": {
+                "x-ms-client-name": "ClientRequestId",
+                "type": "string",
+                "description": "If a client request id header is sent in the request, this header will be present in the response with the same value."
+              },
+              "x-ms-request-id": {
+                "x-ms-client-name": "RequestId",
+                "type": "string",
+                "descript
```

Comment:
I wonder if this should have `"format": "binary"`...

Reply:
Per https://swagger.io/docs/specification/data-models/data-types/, a base64 string should be `"format": "byte"` but I have no idea how well our code generators handle that.

Reply:
Though for what it's worth, our `StorageSharedKeyCredential` expects this key as a base64 string instead of a byte array.